common/djangoapps/track/views/tests/test_segmentio.py

```diff
@@ -389,22 +390,19 @@ def test_video_event(self, name, event_type):
                     },
                     'received_at': datetime.strptime("2014-08-27T16:33:39.100Z", "%Y-%m-%dT%H:%M:%S.%fZ"),
                 },
-            }
-            expected_payload = {
-                'currentTime': 132.134456,
-                'id': 'i4x-foo-bar-baz-some_module',
-                'code': 'mobile'
+                'event': {
+                    'currentTime': 132.134456,
+                    'id': 'i4x-foo-bar-baz-some_module',
+                    'code': 'mobile'
+                }
             }
             if name == 'edx.video.loaded':
-                del expected_payload['currentTime']
+                del expected_event['event']['currentTime']
```

Comment:
This is actually just setting up the expected event properly. The "load_video" event does not include a "currentTime" field in the payload, so we remove it from the expected_event payload.